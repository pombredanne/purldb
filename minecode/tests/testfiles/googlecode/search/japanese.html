


<!DOCTYPE html>
<html>
<head>
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 (function(){function a(b){this.t={};this.tick=function(b,d,e){d=e!=void 0?e:(new Date).getTime();this.t[b]=d};this.tick("start",null,b)}var c=new a;window.jstiming={Timer:a,load:c};try{var f=null;window.chrome&&window.chrome.csi&&(f=Math.floor(window.chrome.csi().pageT));f==null&&window.gtbExternal&&(f=window.gtbExternal.pageT());f==null&&window.external&&(f=window.external.pageT);f&&(window.jstiming.pt=f)}catch(g){};})();

 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"token":null,"assetHostPath":"http://www.gstatic.com/codesite/ph","domainName":null,"assetVersionPath":"http://www.gstatic.com/codesite/ph/1215397193597998757","projectName":null,"projectHomeUrl":null,"absoluteBaseUrl":"http://code.google.com","relativeBaseUrl":"","urlPrefix":null,"loggedInUserEmail":null};
 </script>
 
 
 <title>
 
 
 Google Project Hosting
 </title>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOINDEX,NOARCHIVE">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/1215397193597998757/css/ph_core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/1215397193597998757/css/ph_list.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/1215397193597998757/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 
 .activity-level-High {
 background:no-repeat url(http://www.gstatic.com/codesite/ph/images/activity-level.png) 0 0;
 width: 16px;
 height: 16px;
 }
 .activity-level-Low {
 background:no-repeat url(http://www.gstatic.com/codesite/ph/images/activity-level.png) 0 -16px;
 width: 16px;
 height: 16px;
 }
 .activity-level-Medium {
 background: no-repeat url(http://www.gstatic.com/codesite/ph/images/activity-level.png) -16px 0;
 width: 16px;
 height: 16px;
 }
 .activity-level-None {
 background:no-repeat url(http://www.gstatic.com/codesite/ph/images/activity-level.png) -16px -16px;
 width: 16px;
 height: 16px;
 }

 .labels a:link { color: #080 }
 .labels a:visited { color: #080 }
 .labels a:active { color: #f00 }
 .name { margin-top: 2ex; font-size: 120% }

 </style>
</head>
<body class="">
 <script type="text/javascript">
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 </script>
<div class="headbg">
 <div id="gaia">
 

 <span>
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fhosting%2Fsearch%3Fq%3Djapan&amp;followup=http%3A%2F%2Fcode.google.com%2Fhosting%2Fsearch%3Fq%3Djapan" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>
 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->




 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0">
 <tr style="height: 58px;">
 
 <td id="plogo">
 <a href="/">
 
 <img src="http://www.gstatic.com/codesite/ph/images/code_small.png" alt="Logo">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 </td>
 <td style="white-space:nowrap;">
 
 <form action="/hosting/search">
 <input size="40" name="q" value="japan" type="text">
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 

<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 
<!-- IE -->




<script type="text/javascript">
 function starClick(el, project_name) {
 _CS_toggleStar(
 el, '',
 {'scope': 'projects',
 'user': '_CURRENT_USER',
 'item': project_name,
 'token': codesite_token
 },
 true,
 '',
 '',
 '',
 'star_count-' + project_name,
 'plural-' + project_name);
 }
</script>
<table class="mainhdr" width="100%" border="0" cellspacing="0" cellpadding="0">
<tr><td width="100%"></td>

 <td style="padding:3px 1em; font-weight:normal; white-space:nowrap;">
 Results 1 - 10 of 263
 <span style="font-size: 80%; margin: 1em">
 <a class="buttonify capsule_left mode_button_active"
 href="search?q=japan&amp;start=0">List</a><a
 class="buttonify capsule_right "
 href="search?q=japan&amp;start=0&amp;mode=grid">Grid</a>
 </span>
 </td>

</tr>
</table>
<br/>
<div id="serp">

 
 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/ankidroid/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/ankidroid/logo?cct=1308643200.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 1)"
 href="/p/ankidroid/" style="font-size:medium">ankidroid &nbsp;- &nbsp;Anki on Android</a>
 
 <br/>
 
 <span style="color:#676767">
 
 Activity: <img src="http://www.gstatic.com/codesite/ph/images/cleardot.gif" class="activity-level-High">
 High&nbsp;&nbsp;-&nbsp;
 
 Updated: Today
 &nbsp;-&nbsp;
 Stars: <span id="star_count-ankidroid">143</span>
 </span><br/>
 
 
 AnkiDroid is a  flashcards application for Android.
It is free (and  open source), you can download it from the Android Market or from  here.


  Smart Spaced-Repetition algorithm for efficient me...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:anki">anki</a>, 
 
 <a href="/hosting/search?q=label:ankidroid">ankidroid</a>, 
 
 <a href="/hosting/search?q=label:android">android</a>, 
 
 <a href="/hosting/search?q=label:srs">srs</a>, 
 
 <a href="/hosting/search?q=label:language">language</a>, 
 
 <a href="/hosting/search?q=label:learning">learning</a>, 
 
 <a href="/hosting/search?q=label:memorizing">memorizing</a>, 
 
 <a href="/hosting/search?q=label:japanese">japanese</a>, 
 
 <a href="/hosting/search?q=label:chinese">chinese</a>, 
 
 <a href="/hosting/search?q=label:study">study</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/slim3/">
 <img style="max-width:50px;max-height:55px"
 
 src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 2)"
 href="/p/slim3/" style="font-size:medium">slim3 &nbsp;- &nbsp;Full-stack MVC framework optimized for Google App Engine/Java</a>
 
 <br/>
 
 <span style="color:#676767">
 
 Activity: <img src="http://www.gstatic.com/codesite/ph/images/cleardot.gif" class="activity-level-High">
 High&nbsp;&nbsp;-&nbsp;
 
 Updated: Last 30 days
 &nbsp;-&nbsp;
 Stars: <span id="star_count-slim3">89</span>
 </span><br/>
 
 
 Slim3 is a full-stack MVC framework optimized for Google App Engine/Java.

Our main concept is &quot;Simple&quot; and &quot;Less Is More&quot;.
&quot;Less is more&quot; means simplicity and clarity lead to good design.

80% o...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:Java">Java</a>, 
 
 <a href="/hosting/search?q=label:AppEngine">AppEngine</a>, 
 
 <a href="/hosting/search?q=label:framework">framework</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/dragonnesttranzlation/">
 <img style="max-width:50px;max-height:55px"
 
 src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 3)"
 href="/p/dragonnesttranzlation/" style="font-size:medium">dragonnesttranzlation &nbsp;- &nbsp;Dragon Nest Japan To English Translation Project</a>
 
 <br/>
 
 <span style="color:#676767">
 
 Activity: <img src="http://www.gstatic.com/codesite/ph/images/cleardot.gif" class="activity-level-None">
 None&nbsp;&nbsp;-&nbsp;
 
 Updated: Aug 04, 2010
 &nbsp;-&nbsp;
 Stars: <span id="star_count-dragonnesttranzlation">2</span>
 </span><br/>
 
 
 Dragon Nest Japan To English Translation Project<br/>
 
 <span class="labels">
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/sanguoforciv4/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/sanguoforciv4/logo?cct=1308643200.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 4)"
 href="/p/sanguoforciv4/" style="font-size:medium">sanguoforciv4 &nbsp;- &nbsp;The History of Three Kingdoms (HoTK): A total conversion Mod about Thr...</a>
 
 <br/>
 
 <span style="color:#676767">
 
 Activity: <img src="http://www.gstatic.com/codesite/ph/images/cleardot.gif" class="activity-level-None">
 None&nbsp;&nbsp;-&nbsp;
 
 Updated: May 08, 2010
 &nbsp;-&nbsp;
 Stars: <span id="star_count-sanguoforciv4">6</span>
 </span><br/>
 
 
 



This Mod is a historically based fan Mod that covers the time of Three Kingdoms period of China (184 AD – 280 AD). Wikipedia link: 

Although relatively short, this historical period has b...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:HoTK">HoTK</a>, 
 
 <a href="/hosting/search?q=label:civ4">civ4</a>, 
 
 <a href="/hosting/search?q=label:bts">bts</a>, 
 
 <a href="/hosting/search?q=label:mod">mod</a>, 
 
 <a href="/hosting/search?q=label:TheHistoryofThreeKingdoms">TheHistoryofThreeKingdoms</a>, 
 
 <a href="/hosting/search?q=label:romanceofthreekingdoms">romanceofthreekingdoms</a>, 
 
 <a href="/hosting/search?q=label:ThreeKingdoms">ThreeKingdoms</a>, 
 
 <a href="/hosting/search?q=label:文明4">文明4</a>, 
 
 <a href="/hosting/search?q=label:sanguo">sanguo</a>, 
 
 <a href="/hosting/search?q=label:history">history</a>, 
 
 <a href="/hosting/search?q=label:三国春秋">三国春秋</a>, 
 
 <a href="/hosting/search?q=label:CAR">CAR</a>, 
 
 <a href="/hosting/search?q=label:CivAccelerator">CivAccelerator</a>, 
 
 <a href="/hosting/search?q=label:三国">三国</a>, 
 
 <a href="/hosting/search?q=label:China">China</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/opensocial-jquery/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/opensocial-jquery/logo?cct=1308643200.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 5)"
 href="/p/opensocial-jquery/" style="font-size:medium">opensocial-jquery &nbsp;- &nbsp;OpenSocial jQuery is jQuery based concise JavaScript Library for rapid...</a>
 
 <br/>
 
 <span style="color:#676767">
 
 Activity: <img src="http://www.gstatic.com/codesite/ph/images/cleardot.gif" class="activity-level-Medium">
 Medium&nbsp;&nbsp;-&nbsp;
 
 Updated: Earlier this year
 &nbsp;-&nbsp;
 Stars: <span id="star_count-opensocial-jquery">121</span>
 </span><br/>
 
 
    

 OpenSocial JavaScript API is good,  Lightweight JavaScript API is better, and OpenSocial jQuery is best.



OpenSocial jQ...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:opensocial">opensocial</a>, 
 
 <a href="/hosting/search?q=label:jquery">jquery</a>, 
 
 <a href="/hosting/search?q=label:gadgets">gadgets</a>, 
 
 <a href="/hosting/search?q=label:shindig">shindig</a>, 
 
 <a href="/hosting/search?q=label:jsdeferred">jsdeferred</a>, 
 
 <a href="/hosting/search?q=label:friendconnect">friendconnect</a>, 
 
 <a href="/hosting/search?q=label:oauth">oauth</a>, 
 
 <a href="/hosting/search?q=label:mixi">mixi</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/aedict/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/aedict/logo?cct=1308643200.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 6)"
 href="/p/aedict/" style="font-size:medium">aedict &nbsp;- &nbsp;English-Japanese dictionary for Android</a>
 
 <br/>
 
 <span style="color:#676767">
 
 Activity: <img src="http://www.gstatic.com/codesite/ph/images/cleardot.gif" class="activity-level-High">
 High&nbsp;&nbsp;-&nbsp;
 
 Updated: Last 7 days
 &nbsp;-&nbsp;
 Stars: <span id="star_count-aedict">38</span>
 </span><br/>
 
 
 
A free, open-source english-japanese dictionary for Android which uses Jim Breen&#39;s edict data. Does not require japanese keyboard. Works offline. The dictionary data is downloaded automatically. 
...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:dictionary">dictionary</a>, 
 
 <a href="/hosting/search?q=label:english">english</a>, 
 
 <a href="/hosting/search?q=label:japanese">japanese</a>, 
 
 <a href="/hosting/search?q=label:edict">edict</a>, 
 
 <a href="/hosting/search?q=label:offline">offline</a>, 
 
 <a href="/hosting/search?q=label:kanjidic">kanjidic</a>, 
 
 <a href="/hosting/search?q=label:radicals">radicals</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/furigana-injector/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/furigana-injector/logo?cct=1308643200.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 7)"
 href="/p/furigana-injector/" style="font-size:medium">furigana-injector &nbsp;- &nbsp;Inserts furigana for kanji words (Mozilla Firefox and Google Chrome ex...</a>
 
 <br/>
 
 <span style="color:#676767">
 
 Activity: <img src="http://www.gstatic.com/codesite/ph/images/cleardot.gif" class="activity-level-Medium">
 Medium&nbsp;&nbsp;-&nbsp;
 
 Updated: Yesterday
 &nbsp;-&nbsp;
 Stars: <span id="star_count-furigana-injector">41</span>
 </span><br/>
 
 
 ふりがなインジェクター - Nanoszenie furigany - 振假名植入器 - حاقن الفوريغانا 



Sorry, but due to lack of support I never had a standby server, and finally my orig...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:furigana">furigana</a>, 
 
 <a href="/hosting/search?q=label:mozilla">mozilla</a>, 
 
 <a href="/hosting/search?q=label:xpcom">xpcom</a>, 
 
 <a href="/hosting/search?q=label:nihongo">nihongo</a>, 
 
 <a href="/hosting/search?q=label:振りがな">振りがな</a>, 
 
 <a href="/hosting/search?q=label:フリガナ">フリガナ</a>, 
 
 <a href="/hosting/search?q=label:ふりがな">ふりがな</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/l-echo/">
 <img style="max-width:50px;max-height:55px"
 
 src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 8)"
 href="/p/l-echo/" style="font-size:medium">l-echo &nbsp;- &nbsp;open source echochrome clone</a>
 
 <br/>
 
 <span style="color:#676767">
 
 Activity: <img src="http://www.gstatic.com/codesite/ph/images/cleardot.gif" class="activity-level-None">
 None&nbsp;&nbsp;-&nbsp;
 
 Updated: Jun 19, 2010
 &nbsp;-&nbsp;
 Stars: <span id="star_count-l-echo">11</span>
 </span><br/>
 
 
 Notice: As of 3-20-09, I am no longer developing this.  This is a failure on my part, because of my bad coding practices.  I can no longer maintain and add to my code, so this project is becoming a hu...<br/>
 
 <span class="labels">
 
 
 <a href="/hosting/search?q=label:game">game</a>, 
 
 <a href="/hosting/search?q=label:linux">linux</a>, 
 
 <a href="/hosting/search?q=label:echochrome">echochrome</a>, 
 
 <a href="/hosting/search?q=label:sony">sony</a>, 
 
 <a href="/hosting/search?q=label:clone">clone</a>, 
 
 <a href="/hosting/search?q=label:scej">scej</a>
 
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/maniadb/">
 <img style="max-width:50px;max-height:55px"
 
 src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 9)"
 href="/p/maniadb/" style="font-size:medium">maniadb &nbsp;- &nbsp;iTunes helper which displays album arts of current song by searching m...</a>
 
 <br/>
 
 <span style="color:#676767">
 
 Activity: <img src="http://www.gstatic.com/codesite/ph/images/cleardot.gif" class="activity-level-None">
 None&nbsp;&nbsp;-&nbsp;
 
 Updated: Feb 12, 2010
 &nbsp;-&nbsp;
 Stars: <span id="star_count-maniadb">1</span>
 </span><br/>
 
 
 It gets current song information from iTunes then searches  maniadb.com, Amazon, Amazon Japan, cover.zzlzzl.net, and AlbumArtExchange.com. It shows search results in separate window.

It&#39;s written w...<br/>
 
 <span class="labels">
 
 </span>
 </td>
 </tr>
</table>

 
 
<table>
 <tr>
 <td style="padding-right: 5px;width:50px;">
 <a href="/p/sahanadocsjp/">
 <img style="max-width:50px;max-height:55px"
 
 src="/p/sahanadocsjp/logo?cct=1308643200.0"
 >
 </a>
 </td>
 <td style="width:650px; padding-bottom:10px">
 <a onmousedown="return clk(this, 10)"
 href="/p/sahanadocsjp/" style="font-size:medium">sahanadocsjp &nbsp;- &nbsp;Sahana Eden 日本語 ドキュメント</a>
 
 <br/>
 
 <span style="color:#676767">
 
 Activity: <img src="http://www.gstatic.com/codesite/ph/images/cleardot.gif" class="activity-level-High">
 High&nbsp;&nbsp;-&nbsp;
 
 Updated: Earlier this year
 &nbsp;-&nbsp;
 Stars: <span id="star_count-sahanadocsjp">1</span>
 </span><br/>
 
 
 

Sahana Eden の日本語情報を収集しています。

下記のサイトから、底本を取得しています:

  wiki: 

下記のサイトから、和訳を収集しています:
...<br/>
 
 <span class="labels">
 
 </span>
 </td>
 </tr>
</table>

 
 
 </div>
 <div style="clear:both">&nbsp;</div>


<div style="text-align:center; font-size:medium; white-space:nowrap; margin-top:1em;">
 
 
 
	1
 
 
 
 <a href="search?q=japan&amp;filter=0&amp;mode=&amp;start=10">2</a>
 
 
 
 <a href="search?q=japan&amp;filter=0&amp;mode=&amp;start=20">3</a>
 
 
 
 <a href="search?q=japan&amp;filter=0&amp;mode=&amp;start=30">4</a>
 
 
 
 <a href="search?q=japan&amp;filter=0&amp;mode=&amp;start=40">5</a>
 
 
 
 <a href="search?q=japan&amp;filter=0&amp;mode=&amp;start=50">6</a>
 
 
 
 <a href="search?q=japan&amp;filter=0&amp;mode=&amp;start=60">7</a>
 
 
 
 <a href="search?q=japan&amp;filter=0&amp;mode=&amp;start=70">8</a>
 
 
 
 <a href="search?q=japan&amp;filter=0&amp;mode=&amp;start=80">9</a>
 
 
 
 <a href="search?q=japan&amp;filter=0&amp;mode=&amp;start=90">10</a>
 
 
 <a href="search?q=japan&amp;filter=0&amp;mode=&amp;start=10">Next <b>&rsaquo;</b></a>
</div>
<br/>
<br/>
<div style="background:#F0F7F9; border-top:1px solid #6B90DA; border-bottom:1px solid #6B90DA; text-align:center;">
 <br/>
 <form action="/hosting/search">
 <input type="text" size="30" name="q" value="japan">
 <input type="hidden" name="mode" value="list">
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 <br/>
</div>
<script type="text/javascript">
function clk(el, cd) {
 if (document.images) {
 (new Image()).src = "/hosting/search?sa=T&ct=res&cd=" + cd + "&url="
 + escape(el.href) + "&q=japan";
 }
 return true;
}
</script>
 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/1215397193597998757/js/ph_core.js"></script>
 
 
 
 
 <script type="text/javascript" src="/js/codesite_product_dictionary_ph.pack.04102009.js"></script>
</div> 
<div id="footer" dir="ltr">
 <div class="text">
 &copy;2011 Google -
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>
 
 


 
 
 <script type="text/javascript">_CS_reportToCsi();</script>
 
 </body>
</html>

