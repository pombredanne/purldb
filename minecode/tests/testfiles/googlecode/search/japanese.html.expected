[
  [
    "http://code.google.com/hosting/search?q=japan&filter=0&mode=&start=10", 
    "http://code.google.com/hosting/search?q=japan&filter=0&mode=&start=20", 
    "http://code.google.com/hosting/search?q=japan&filter=0&mode=&start=30", 
    "http://code.google.com/hosting/search?q=japan&filter=0&mode=&start=40", 
    "http://code.google.com/hosting/search?q=japan&filter=0&mode=&start=50", 
    "http://code.google.com/hosting/search?q=japan&filter=0&mode=&start=60", 
    "http://code.google.com/hosting/search?q=japan&filter=0&mode=&start=70", 
    "http://code.google.com/hosting/search?q=japan&filter=0&mode=&start=80", 
    "http://code.google.com/hosting/search?q=japan&filter=0&mode=&start=90", 
    "http://code.google.com/hosting/search?q=label:android", 
    "http://code.google.com/hosting/search?q=label:anki", 
    "http://code.google.com/hosting/search?q=label:ankidroid", 
    "http://code.google.com/hosting/search?q=label:appengine", 
    "http://code.google.com/hosting/search?q=label:bts", 
    "http://code.google.com/hosting/search?q=label:car", 
    "http://code.google.com/hosting/search?q=label:china", 
    "http://code.google.com/hosting/search?q=label:chinese", 
    "http://code.google.com/hosting/search?q=label:civ4", 
    "http://code.google.com/hosting/search?q=label:civaccelerator", 
    "http://code.google.com/hosting/search?q=label:clone", 
    "http://code.google.com/hosting/search?q=label:dictionary", 
    "http://code.google.com/hosting/search?q=label:echochrome", 
    "http://code.google.com/hosting/search?q=label:edict", 
    "http://code.google.com/hosting/search?q=label:english", 
    "http://code.google.com/hosting/search?q=label:framework", 
    "http://code.google.com/hosting/search?q=label:friendconnect", 
    "http://code.google.com/hosting/search?q=label:furigana", 
    "http://code.google.com/hosting/search?q=label:gadgets", 
    "http://code.google.com/hosting/search?q=label:game", 
    "http://code.google.com/hosting/search?q=label:history", 
    "http://code.google.com/hosting/search?q=label:hotk", 
    "http://code.google.com/hosting/search?q=label:japanese", 
    "http://code.google.com/hosting/search?q=label:java", 
    "http://code.google.com/hosting/search?q=label:jquery", 
    "http://code.google.com/hosting/search?q=label:jsdeferred", 
    "http://code.google.com/hosting/search?q=label:kanjidic", 
    "http://code.google.com/hosting/search?q=label:language", 
    "http://code.google.com/hosting/search?q=label:learning", 
    "http://code.google.com/hosting/search?q=label:linux", 
    "http://code.google.com/hosting/search?q=label:memorizing", 
    "http://code.google.com/hosting/search?q=label:mixi", 
    "http://code.google.com/hosting/search?q=label:mod", 
    "http://code.google.com/hosting/search?q=label:mozilla", 
    "http://code.google.com/hosting/search?q=label:nihongo", 
    "http://code.google.com/hosting/search?q=label:oauth", 
    "http://code.google.com/hosting/search?q=label:offline", 
    "http://code.google.com/hosting/search?q=label:opensocial", 
    "http://code.google.com/hosting/search?q=label:radicals", 
    "http://code.google.com/hosting/search?q=label:romanceofthreekingdoms", 
    "http://code.google.com/hosting/search?q=label:sanguo", 
    "http://code.google.com/hosting/search?q=label:scej", 
    "http://code.google.com/hosting/search?q=label:shindig", 
    "http://code.google.com/hosting/search?q=label:sony", 
    "http://code.google.com/hosting/search?q=label:srs", 
    "http://code.google.com/hosting/search?q=label:study", 
    "http://code.google.com/hosting/search?q=label:thehistoryofthreekingdoms", 
    "http://code.google.com/hosting/search?q=label:threekingdoms", 
    "http://code.google.com/hosting/search?q=label:xpcom", 
    "http://code.google.com/hosting/search?q=label:\u3075\u308a\u304c\u306a", 
    "http://code.google.com/hosting/search?q=label:\u30d5\u30ea\u30ac\u30ca", 
    "http://code.google.com/hosting/search?q=label:\u4e09\u56fd", 
    "http://code.google.com/hosting/search?q=label:\u4e09\u56fd\u6625\u79cb", 
    "http://code.google.com/hosting/search?q=label:\u632f\u308a\u304c\u306a", 
    "http://code.google.com/hosting/search?q=label:\u6587\u660e4"
  ], 
  [
    {
      "url": "http://code.google.com/p/ankidroid/", 
      "last_updated": "2014-02-02T00:00:00+00:00", 
      "stars": 143
    }, 
    {
      "url": "http://code.google.com/p/slim3/", 
      "last_updated": "2014-01-03T00:00:00+00:00", 
      "stars": 89
    }, 
    {
      "url": "http://code.google.com/p/dragonnesttranzlation/", 
      "last_updated": "2010-08-04T00:00:00+00:00", 
      "stars": 2
    }, 
    {
      "url": "http://code.google.com/p/sanguoforciv4/", 
      "last_updated": "2010-05-08T00:00:00+00:00", 
      "stars": 6
    }, 
    {
      "url": "http://code.google.com/p/opensocial-jquery/", 
      "last_updated": "2014-01-01T00:00:00+00:00", 
      "stars": 121
    }, 
    {
      "url": "http://code.google.com/p/aedict/", 
      "last_updated": "2014-01-26T00:00:00+00:00", 
      "stars": 38
    }, 
    {
      "url": "http://code.google.com/p/furigana-injector/", 
      "last_updated": "2014-02-01T00:00:00+00:00", 
      "stars": 41
    }, 
    {
      "url": "http://code.google.com/p/l-echo/", 
      "last_updated": "2010-06-19T00:00:00+00:00", 
      "stars": 11
    }, 
    {
      "url": "http://code.google.com/p/maniadb/", 
      "last_updated": "2010-02-12T00:00:00+00:00", 
      "stars": 1
    }, 
    {
      "url": "http://code.google.com/p/sahanadocsjp/", 
      "last_updated": "2014-01-01T00:00:00+00:00", 
      "stars": 1
    }
  ]
]